src/number/n_GF.jl

```diff
@@ -384,10 +384,10 @@ system with given characteristic, degree and string. If this is not the
 desired behaviour, one can pass `false` for the optional `cached` parameter.
 """
 function FiniteField(p::Int, n::Int, S::String; cached=true)
-   n >= 16 || p >= 2^8 && throw(DomainError())
```

Comment:
`n >= 16` here was presumably meant to lead to an error; but this case should be handled by `n*log(p) >= 20*log(2) || p^n >= 2^16` below (I think).

Reply:
Is this tested?

Reply:
I tested for `2^16`, and without overflow, for `p>2` and `n>16`, we have `p^n >= 2^16` which errors. And I think `n*log(p) >= 20*log(2)` takes care that we also error in case `p^n` overflows (it basically ensures that `p^n < 2^20` in floating point, in other words `p^n` doesn't overflow with integers). I believe @wbhart wrote these check, maybe you can comment on this?

Reply:
Values were provided by Hans.

Reply:
By test I meant: Is there a test covering that this throws?

Reply:
> By test I meant: Is there a test covering that this throws?

It's what I understood, I tested that this throws for `2^16` (and for `2^64`) only. I could add a test for some random values too (but this won't be a proof). I can as well add back the check `n >= 16` above if we are not confident enough that this is already covered.